packages/owl-bot/src/owl-bot.ts

```diff
@@ -76,15 +78,29 @@ export = (privateKey: string | undefined, app: Probot, db?: Db) => {
       if (!installation) {
         throw Error(`no installation token found for ${head.repo.full_name}`);
       }
+      const hasRunLabel = !!context.payload.pull_request.labels.filter(
+        l => l.name === OWLBOT_RUN_LABEL
+      ).length;
+
       // If the pull request is from a fork, the label "owlbot:run" must be
       // added by a maintainer to trigger the post processor.
```

Comment:
Which line of code confirms that the label was added by a maintainer?

Reply:
adding a label requires >= "Triage" permissions